docs/_markbind/layouts/ug-sitenav.md

```diff
@@ -7,7 +7,7 @@
   {name: "Appendices"},
   {level: 2, name: "CLI syntax reference", link: "ug/cli.html"},
   {level: 2, name: "Config files format", link: "ug/configFiles.html"},
-  {level: 2, name: "Advanced syntax: `author-config.csv`", link: "ug/authorConfigSyntax.html"}, 
+  {level: 2, name: "Advanced syntax: `author-config.csv`", link: "ug/authorConfigSyntax.html"},
```

Comment:
Seems like these changes are from fixing the lint warnings? Can these be moved to a separate PR as they're not part of the original PR?

Reply:
ok, done!